### PR TITLE
NGAlert: Add integration tests for remaining notification channels

### PR DESCRIPTION
**Pull request by codesome**
**What this PR does / why we need it**:

This PR adds integration tests for all remaining channels that were skipped before (except email, that needs different handling).

### CODE REVIEW

File: pkg/services/ngalert/notifier/channels/line.go [hunk 1]
```diff
@@ -18,7 +18,7 @@ import (
 	old_notifiers "github.com/grafana/grafana/pkg/services/alerting/notifiers"
 )
 
-const (
+var (
```
Comment (papagian) [inline pkg/services/ngalert/notifier/channels/line.go:21]: Is this necessary? I don't think that this value ever changes.

File: pkg/tests/api/alerting/api_notification_channel_test.go [hunk 2]
```diff
@@ -220,31 +250,60 @@ func (nc *mockNotificationChannel) matchesExpNotifications(exp map[string][]stri
...
+			case "googlechat_recv/googlechat_test":
+				// "Grafana v | 25 May 21 17:44 IST"
+				r1 = regexp.MustCompile(`.*"text"\s*:\s*"(Grafana v[^"]+)"`)
+			case "victorops_recv/victorops_test":
+				// It has a time component "timestamp".
+				r1 = regexp.MustCompile(`.*"timestamp"\s*:\s*([0-9]{10})`)
+			case "v1/alerts":
+				// It has a changing time fields.
+				r1 = regexp.MustCompile(`.*"startsAt"\s*:\s*"([^"]+)"`)
+				r2 = regexp.MustCompile(`.*"UpdatedAt"\s*:\s*"([^"]+)"`)
```
Comment (papagian) [inline pkg/tests/api/alerting/api_notification_channel_test.go:278]: out of curiosity, why it's not `updatedAt`?

**Review by papagian — APPROVED** [review]

File: pkg/tests/api/alerting/api_notification_channel_test.go (hunk 2 above)
Comment (codesome) [reply pkg/tests/api/alerting/api_notification_channel_test.go:278]: No idea, it comes from Prometheus Alertmanager

File: pkg/services/ngalert/notifier/channels/line.go (hunk 1 above)
Comment (codesome) [reply pkg/services/ngalert/notifier/channels/line.go:21]: `channels.LineNotifyURL` is actually being changed in the integration test

**Pull request merged**